### ISSUE TITLE
Amazing project, is it still being developed? 

### ISSUE THREAD

**Issue by Delvien** [BlackCapCoder/compton#15]
See title. 

Thanks for all your work!

**Comment by BlackCapCoder** [comment]
Not really. I got it to state where it works for me and shared it in hopes that I'd be useful to others; I didn't expect it to blow up the way it did!
The compton codebase is a mess so the barrier to make even tiny changes is huge. I developed this fork over hours of tedious experimentation- brute force essentially.
Eventually I would like to either refactor the codebase to make it maintainable, or to make my changes less hacky so that I can send a PR, but I lack motivation because "it works on my machine" (tm).


Unrelated to the question: To anyone wanting to hack on compton on their own- I can highly recommend implementing your own little composite manager from scratch first. You are going to read through a bunch of examples and projects, and you are going to recognize blocks of them in compton, just with more gunk. This greatly helps when orientating yourself in the code!

**Comment by Avi-D-coder** [comment]
@yshui has a very active fork [yshui/compton](https://github.com/yshui/compton)
